src/servers/src/http/arrow_result.rs

```diff
@@ -127,7 +124,7 @@ impl IntoResponse for ArrowResponse {
                 ),
                 (
                     &GREPTIME_DB_HEADER_FORMAT,
-                    HeaderValue::from_static("ARROW"),
+                    HeaderValue::from_static(ResponseFormat::Arrow.as_str()),
```

Comment:
It will now use `arrow` instead of `ARROW`, is it intentional? Same as `CSV` and `TABLE`.

Reply:
This is commented at https://github.com/GreptimeTeam/greptimedb/pull/3548#discussion_r1531518349. Please take a look and reply there.